Update runtime to 47

FakeShemp opened the pull request.

FakeShemp committed:
Update runtime to 47

FakeShemp committed:
Update shared-modules

muelli commented:
bot, build

muelli commented:
```
windowmanager.c: In function ‘gtksharp_get_gdk_net_client_list’:
windowmanager.c:113:26: error: assignment to ‘gpointer’ {aka ‘void *’} from ‘long int’ makes pointer from integer without a cast [-Wint-conversion]
  113 |                 list [i] = data [i];
      |                          ^
In file included from /usr/include/glib-2.0/glib.h:64,
                 from /app/include/gtk-2.0/gdk/gdktypes.h:36,
                 from /app/include/gtk-2.0/gdk/gdkscreen.h:32,
                 from windowmanager.c:27:
```

muelli committed:
Patching gtk-sharp to build with more aggressive compiler flags

windowmanager.c: In function ‘gtksharp_get_gdk_net_client_list’:
windowmanager.c:113:26: error: assignment to ‘gpointer’ {aka ‘void *’} from ‘long int’ makes pointer from integer without a cast [-Wint-conversion]
  113 |                 list [i] = data [i];
      |                          ^
In file included from /usr/include/glib-2.0/glib.h:64,
                 from /app/include/gtk-2.0/gdk/gdktypes.h:36,
                 from /app/include/gtk-2.0/gdk/gdkscreen.h:32,
                 from windowmanager.c:27:

muelli commented:
@FakeShemp  merge https://github.com/flathub/com.github.afrantzis.Bless/pull/10 into this branch then the runtime can be bumped.

FakeShemp committed:
Merge pull request #10 from muelli/FakeShemp-patch-1

Patching gtk-sharp to build with more aggressive compiler flags
